### PR TITLE
Allow camera poses to be saved if the octree is served locally.

### PR DESCRIPTION
**Pull request by SirVer**

### CODE REVIEW

File: sdl_viewer/src/lib.rs [hunk 1]
```diff
@@ -271,6 +277,41 @@ impl PointCloudRenderer {
...
+    states: Vec<camera::State>,
+}
+
+fn save_camera(index: usize, pose_path: &Option<PathBuf>, camera: &Camera) {
+    if pose_path.is_none() {
+        println!("Not serving from a local directory. Cannot save camera.");
+        return;
+    }
+    assert!(index < 10);
+    let mut states = ::std::fs::read_to_string(pose_path.as_ref().unwrap()).and_then(|data| {
```
Comment (pdubroy) [inline sdl_viewer/src/lib.rs]: Nit: `use std::fs;`, as you do for cmp, io, thread, etc. and then just `fs::read_to_string` here.

Comment (SirVer) [reply sdl_viewer/src/lib.rs]: Will do in the followup.

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Pull request merged**